gdx/src/com/badlogic/gdx/graphics/g3d/particles/ParticleShader.java

```diff
@@ -316,7 +316,7 @@ protected void bindMaterial (final Renderable renderable) {
 		for (final Attribute attr : currentMaterial) {
 			final long t = attr.type;
 			if (BlendingAttribute.is(t)) {
-				context.setBlending(true, ((BlendingAttribute)attr).sourceFunction, ((BlendingAttribute)attr).destFunction);
```

Comment:
I think it would be better to pass all properties. Some may want to have separate blending for their particles.

other than that, the PR looks good to me. You also have to resolve conflicts and we're good to go.